### PR TITLE
fix debugger initialization

### PR DESCRIPTION
**Pull request by devoncarew**
Fix an issue in the debugger initialization; if it wasn't the selected page on app open, it wouldn't properly recognize the app under debug.

### CODE REVIEW

**Comment by devoncarew** [comment]
@jacob314 @kenzieschmoll

File: lib/debugger/debugger.dart [hunk 1]
```diff
@@ -243,6 +237,25 @@ class DebuggerScreen extends Screen {
...
   }
 
+  @override
+  void entering() {
+    if (!_inited) {
+      _init();
+    }
+  }
+
+  void _init() {
```
Comment (jacob314) [inline lib/debugger/debugger.dart]: nit: avoid abrev

File: lib/debugger/debugger.dart [hunk 2]
```diff
@@ -34,18 +34,14 @@ import '../ui/primer.dart';
 class DebuggerScreen extends Screen {
   DebuggerScreen()
       : super(name: 'Debugger', id: 'debugger', iconClass: 'octicon-bug') {
+    debuggerState = new DebuggerState();
```
Comment (jacob314) [inline lib/debugger/debugger.dart]: nit: make debuggerState final now that it can be.

Comment (devoncarew) [reply lib/debugger/debugger.dart]: done

File: lib/debugger/debugger.dart (hunk 1 above)
Comment (devoncarew) [reply lib/debugger/debugger.dart]: done

**Pull request merged**